it('should throw an error if already resolving', function (next) {
var resolver = new Resolver('foo'),
ok;
